app/views/items/_show_bottom_metadata.html.erb

```diff
@@ -19,6 +19,14 @@
     <% end %>
 
     <li><%= metadata(@res, "Document ID", "identifier", link: false) %></li>
+    
+     <% if @res["type"].present? %>    
+    <li>
+      <strong>Nature of source text: </strong> 
+      <%= @res["type"] %>
+      (ALS: autograph letter signed; ADS: autograph draft signed; ADU: autograph draft unsigned; TLS: typed letter signed; TDS: typed draft signed; TDU: typed draft unsigned; TCS: typed copy signed; TCU: typed copy unsigned; ATR: autograph telegram received; TTR: typed telegram received; EXT: external source. Nomenclature derived from <em>To Be an Author: Letters of Charles W. Chesnutt</em>, ed. Joseph R. McElrath, Jr. and Robert C. Leitz, III [Princeton University Press, 1997], xv.)
```

Comment:
This key seems a little text heavy to me, but maybe you have had discussions about it.